store/postgres/src/chain_store.rs

```diff
@@ -939,6 +936,63 @@ mod data {
             }
         }
 
+        pub(super) fn get_calls_in_block(
+            &self,
+            conn: &PgConnection,
+            block_ptr: BlockPtr,
+        ) -> Result<Vec<CachedEthereumCall>, Error> {
+            let block_num = block_ptr.block_number();
+
+            let rows = match self {
+                Storage::Shared => {
+                    use public::{eth_call_cache as cache, eth_call_meta as meta};
+
+                    cache::table
+                        .inner_join(meta::table)
+                        .select((
+                            cache::id,
+                            cache::return_value,
+                            cache::contract_address,
+                            meta::accessed_at,
+                        ))
+                        .filter(cache::block_number.eq(block_num))
+                        .order(cache::contract_address)
+                        .get_results::<(Vec<u8>, Vec<u8>, Vec<u8>, NaiveDate)>(conn)?
+                }
+                Storage::Private(Schema {
+                    call_cache,
+                    call_meta,
+                    ..
+                }) => call_cache
+                    .table()
+                    .select((
+                        call_cache.id(),
+                        call_cache.return_value(),
+                        call_cache.contract_address(),
+                        call_meta.accessed_at(),
+                    ))
+                    .inner_join(
+                        call_meta.table().on(call_meta
+                            .contract_address()
+                            .eq(call_cache.contract_address())),
+                    )
```

Comment:
It might be that this join will be very slow in practice, since `call_cache` doesn't have an index on `contract_address`; we'll see how it performs, since there's also a query plan where the lookup by `block_number` happens first and then `call_meta` gets joined along its pk index. If things go wrong, we might have to drop reporting `accessed_at`.

Reply:
Thanks for noticing. Let's keep an eye on this in case performance is unacceptable.

Reply:
On second thought, we don't need the last access timestamp, so we can avoid the `JOIN` at all. https://github.com/graphprotocol/graph-node/pull/3343/commits/f4a7ff9b1f67654d51452d18491ba4355e61f75a fixes this.